sample_custom_component2/main.go

```diff
@@ -0,0 +1,222 @@
+package main
```

Comment:
Let's move these sample components under the Test directory as they are not part of the main line code.

Reply:
Will take renaming and movement changes at last iteration to keep it easy for reviewers.